### PR TITLE
Create a release branch for 2.0.0

### PR DESCRIPTION
**Pull request by jonathancouchman**
Updates README to reflect release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,13 @@ All source files are UTF-8 encoded with Unix line-endings (``LF``, ``\n``). They
 
 We maintain a style guide for the Currencycloud API documentation. See [STYLE_GUIDE.md](STYLE_GUIDE.md).
 
+## Deployed API
+The currently deployed API will be the latest tagged release version of currencycloud-swagger.
```
Comment (derikvercueil) [inline README.md]: I would reword as follows: 

"The current version of the API is the latest tagged release" - and probably add a hyperlink to releases for convenience...

File: README.md [hunk 2]
```diff
@@ -30,6 +30,13 @@ All source files are UTF-8 encoded with Unix line-endings (``LF``, ``\n``). They
 
 We maintain a style guide for the Currencycloud API documentation. See [STYLE_GUIDE.md](STYLE_GUIDE.md).
 
+## API Versions
+The current version of the API is the [latest tagged release](https://github.com/CurrencyCloud/currencycloud-swagger/releases/latest). 
+This version is the valid documentation for the current Currencycloud API. The API definition described in the master branch may differ
+from this and, as such, should not be used directly.
```
Comment (derikvercueil) [inline README.md]: Carriage return...

Comment (jonathancouchman) [reply README.md]: I think it is OK, formats alright in mark down
